src/NHPersistenceCompatibilityTests/Common/NugetHelper.cs

```diff
@@ -27,10 +30,25 @@ public IEnumerable<string> GetPossibleVersionsFor(string packageName, string min
 
         internal void DownloadPackageTo(string packageName, string version, string location)
         {
-            var repo = PackageRepositoryFactory.Default.CreateRepository(packageSource);
+            try
+            {
+                InstallPackageFromSource(packageSource, packageName, version, location);
+            }
+            catch (Exception e)
```

Comment:
This catch-all and install from alternative source seems a bit random? any specific exception we can catch? In which condition might this fail?